tools/importer/rules/marquee.js

```diff
@@ -0,0 +1,104 @@
+/* global WebImporter */
+export default function createMarqueeBlocks(main, document) {
+  const marquees = document.querySelectorAll('#marquee');
+
+  // fast return
+  if (!marquees) {
+    return;
+  }
+
+  marquees.forEach((marquee) => {
+    // Create table for marquee
+    const cells = [['Marquee']];
+
+    // Check if marquee has background video
+    if (marquee.querySelector('.has-video.dexter-Background')) {
+      const backgroundVideoURL = marquee.querySelector('.has-video.dexter-Background').querySelector('video').querySelector('source').getAttribute('src');
+      const bgvideoa = document.createElement('a');
+      bgvideoa.innerHTML = backgroundVideoURL;
+      bgvideoa.setAttribute('href', backgroundVideoURL);
+      cells.push([bgvideoa]);
+    }
```

Comment:
Leverage the css selectors:
```suggestion
    const videoSource = marquee.querySelector('.has-video.dexter-Background video source');
    if (videoSource) {
      const src = videoSource.getAttribute('src');
      const a = document.createElement('a');
      a.innerHTML = src;
      a.setAttribute('href', src);
      cells.push([a]);
    }
```

Simpler and easier to read, no ?

Reply:
Done